packages/calcite-components/package.json

```diff
@@ -67,10 +67,11 @@
     "util:copy-assets": "npm run util:copy-icons",
     "util:copy-icons": "cpy \"../../node_modules/@esri/calcite-ui-icons/js/*.json\" \"./src/components/icon/assets/\" --flat",
     "util:generate-t9n-docs-json": "tsx support/generateT9nDocsJSON.ts",
+    "util:generate-t9n-manifest": "tsx support/generateT9nManifest.ts",
```

Comment:
Should this and the script file use `update` instead of `generate`? The t9n manifest file wouldn't normally be deleted, right?

Reply:
Good catch. lets use `updateT9nManifest` naming.